src/CMakeLists.txt

```diff
@@ -1,161 +0,0 @@
-#
```

Comment:
I'm really surprised at removing this! Isn't cmake idiom usually to have a file per directory? (I honestly don't know, I still feel like I barely understand the whole convoluted mess that is cmake :)

Reply:
It was never a requirement and keeping things in as few files as possible could help with readability. Furthermore, the library definition itself I think should go in the root to be visible.